image/fs_test.go

```diff
@@ -208,7 +208,7 @@ func TestFSWalker(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	testWalker(t, fs)
+	testWalk(t, fs)
```

Comment:
I'm not sure this is correct; it's testing a _walker_ (that "walks")? In any case, I'm not sure we gain much by renaming these